app/views/training/index.html.haml

```diff
@@ -1,22 +1,12 @@
 - content_for :before_title, "Training Libraries - "
 .container.training
   %h1 Training Libraries
-  %ul.training-libraries.no-bullets.no-margin
-    - defocus_class = @focused_library_slug ? 'training-library-defocus' : ''
-    - @libraries.each do |library|
-      - next if library.exclude_from_index?
-      - focus_class = @focused_library_slug == library.slug ? 'training-library-focus' : defocus_class
-      %li{class: "training-libraries__individual-library no-left-margin #{focus_class}"}
-        %a.action-card.action-card-index{href: "/training/#{library.slug}"}
-          %header.action-card-header
-            %h3.action-card-title
-              = library.translated_name
-            %span.icon-container
-              %i.action-card-icon.icon.icon-rt_arrow
-        .action-card-text
-          %h3 Included Modules:
-          %ul
-            - library.categories.pluck('modules').flatten.each do |training_module|
-              %li
-                %a{href: "/training/#{library.slug}/#{training_module['slug']}", target: "_blank"}
-                  = training_module['name']
+  .search-bar{style: 'position:relative'}
+    = form_tag("/training", method: "get") do
+      = text_field_tag(:search_training, '', placeholder: t('training.search_training_resources'), style: 'width: 100%; height: 3rem;font-size: 15px')
+      %button{type: 'submit', name: 'glass_training', style: 'position: absolute; right: 20px; top: 10px'}
```

Comment:
Why is this called 'glass_training'?

Reply:
Okay. I think `query` or `search` would be a better name, so that it's obvious the resulting URL what that parameter is for.

Reply:
We did not really need a name here after all :). An id is better as it do not appears in the ur.l